featuretools/tests/primitive_tests/test_transform_features.py

```diff
@@ -153,45 +154,47 @@ def pd_simple_es():
     })
 
     es = ft.EntitySet('equal_test')
-    es.entity_from_dataframe('values', df, index='id')
+    es.add_dataframe('values', df, index='id')
```

Comment:
The kwarg order got changed on `add_dataframe`, so I think the dataframe needs to come first and then the name string.